<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhotoInfo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="imageLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Файл:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string>/media/foto/Тольятти/Свадьба_Аллы_Палчевской_Александра_2015/32cf_волга/JPEG/IMG_8982.jpg</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pB_Analise">
        <property name="text">
         <string>&amp;Выбор фото</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Выбор фильтра свойств файла:</string>
        </property>
        <property name="buddy">
         <cstring>comboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Без Фильтра</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Только IPTC. Встроенный фильтр</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Только XMP. Встроенный фильтр</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Только EXIF. Встроенный фильтр</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Самое важное CR2. Пользовательский фильтр</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>&amp;Редактор Фильтров</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pB_Analise</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ph_select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ph_analise_()</slot>
  <slot>ph_analise_exiv()</slot>
  <slot>ph_select()</slot>
 </slots>
</ui>
